dex/testing/btc/harness.sh

```diff
@@ -202,20 +211,30 @@ tmux send-keys -t $SESSION:2 "./beta generatetoaddress 110 ${BETA_MINING_ADDR}${
 # Setup the gamma wallet
 ################################################################################
 
-# Create  wallet, encrypt it, and send it some bitcoin, mining some blocks too.
+# Create and encrypt the gamma wallet
 echo "Creating the gamma wallet"
 tmux send-keys -t $SESSION:2 "./alpha createwallet gamma${WAIT}" C-m\; wait-for donebtc
 tmux send-keys -t $SESSION:2 "./gamma sethdseed true ${GAMMA_WALLET_SEED}${WAIT}" C-m\; wait-for donebtc
 tmux send-keys -t $SESSION:2 "./gamma getnewaddress${WAIT}" C-m\; wait-for donebtc
 tmux send-keys -t $SESSION:2 "./gamma getnewaddress \"\" \"legacy\"${WAIT}" C-m\; wait-for donebtc
-echo "Encrypting wallets (may take a minute)"
+echo "Encrypting the gamma wallet (may take a minute)"
 tmux send-keys -t $SESSION:2 "./gamma encryptwallet ${WALLET_PASSWORD}${WAIT}" C-m\; wait-for donebtc
-tmux send-keys -t $SESSION:2 "./beta encryptwallet ${WALLET_PASSWORD}${WAIT}" C-m\; wait-for donebtc
```

Comment:
If I get this correctly, the intent for encrypting the beta wallet is to enable multi-wallet trading using the btc harness wallets. I've added a second wallet that can be used for trading, thus this shouldn't be necessary anymore?